src/lib/core/coio.c

```diff
@@ -88,32 +88,34 @@ coio_connect_addr(const struct sockaddr *addr, socklen_t len, ev_tstamp timeout)
 	return -1;
 }
 
-void
+int
 coio_fill_addrinfo(struct addrinfo *ai_local, const char *host,
```

Comment:
This function is not in the header file. I think we should add it in coio.h

Reply:
No need to have it in coio.h, because it isn't used outside coio.c. I marked it static instead.